warehouse/docker-compose.test.yml

```diff
@@ -16,11 +16,13 @@ services:
       retries: 25
   wh-transformer:
     container_name: wh-transformer
-    image: rudderlabs/rudder-transformer:latest
+    image: rudderlabs/develop-rudder-transformer:master
     ports:
       - "9090"
+    environment:
+      - NUM_PROCS=1
     healthcheck:
```

Comment:
Once we merged [this](https://github.com/rudderlabs/rudder-transformer/pull/1582) PR, we would not be needing this right?